R/prep_data.R

```diff
@@ -139,17 +162,18 @@ prep_data <- function(d,
                       convert_dates = TRUE,
                       impute = TRUE,
                       collapse_rare_factors = TRUE,
-                      center = FALSE,
-                      scale = FALSE,
+                      PCA = FALSE,
```

Comment:
This makes sense from a coding perspective, but it looks strange in the docs. Can you leave these as FALSE and make the changes under the hood?